include/powsybl/iidm/VariantManager.hpp

```diff
@@ -44,8 +44,12 @@ class VariantManager {
 
     void cloneVariant(const std::string& sourceVariantId, const std::string& targetVariantId);
 
+    void cloneVariant(const std::string& sourceVariantId, const std::string& targetVariantId, bool mayOverwrite);
+
     void cloneVariant(const std::string& sourceVariantId, const std::initializer_list<std::string>& targetVariantIds);
 
+    void cloneVariant(const std::string& sourceVariantId, const std::initializer_list<std::string>& targetVariantIds, bool mayOverwrite);
+
```

Comment:
To be discussed: do we add an overload, or do we use default parameters.

Reply:
=> updated to use default parameters